_health-care/form.html

```diff
@@ -0,0 +1,135 @@
+---
+title: Health Care Form Test
+permalink: /health-care/form.html
+template: 1-topic-landing
+---
+
+{% if site.domain != "staging.vets.gov" %}
+<div class="main" role="main">
+  <div class="section primary">
+    <div class="row">
+      <div class="small-12 columns">
+        <h4>This page not in production yet.</h4>
+      </div>
+    </div>
+  </div>
+</div>
+{% else %}
+<style>
+  .tabs.vertical {
+    width: 100%;
+    max-width: 100%;
+  }
+  div.form-section {
+    padding: 2rem;
+    margin: 0 0 2rem;
+    border: 1px solid #eee; 
+  }
+</style>
```

Comment:
We should remove this inline css before this goes to production, but I'm ok with it for now since we probably want to not override all of these things. Happy to make a design pass or two later.